### PR TITLE
Refactor TREC-COVID Round 3 baselines (add RF run)

### PR DESCRIPTION
**Pull request by lintool**
Refactored to simplify replication; updated docs and results as well.

```
$ python src/main/python/trec-covid/download_indexes.py --date 2020-05-19
$ python src/main/python/trec-covid/generate_round3_baselines.py
```

### CODE REVIEW

**Comment by lintool** [comment]
BTW, I'm aware that there's still a fair bit of code duplication. I'm trying to get the results correct first, and then will circle back to refactor.

File: docs/experiments-covid.md [hunk 1]
```diff
@@ -3,13 +3,14 @@
...
+These are runs that can be easily replicated with Anserini, from pre-built indexes available [here](experiments-cord19.md#pre-built-indexes-all-versions) (version from 2020/06/19, which is the official corpus used in round 4).
+They were prepared _for_ round 4 (for participants who wish to have a baseline run to rerank); to provide a sense of effectiveness, we present evaluation results with the cumulative qrels from rounds 1, 2, and 3 ([`qrels_covid_d3_j0.5-3.txt`](https://ir.nist.gov/covidSubmit/data/qrels-covid_d3_j0.5-3.txt) provided by NIST, stored in our repo as [`qrels.covid-round3-cumulative.txt`](../src/main/resources/topics-and-qrels/qrels.covid-round3-cumulative.txt)).
 
-|    | index     | field(s)                        | nDCG@10 | Judged@10 | Recall@1000 | run file | checksum |
-|---:|:----------|:--------------------------------|--------:|----------:|------------:|:---------|----------|
+|    | index     | field(s)                        | nDCG@10 | J@10 | R@1k | run file | checksum |
```
Comment (rodrigonogueira4) [inline docs/experiments-covid.md:12]: not super important, but I think it is better to not abbreviate "Judged@10" as it is not a common metric

File: src/main/python/trec-covid/download_indexes.py [hunk 2]
```diff
@@ -17,55 +17,29 @@
...
 
 
 all_indexes = {
-    '2020-05-19': ['https://www.dropbox.com/s/3ld34ms35zfb4m9/lucene-index-cord19-abstract-2020-05-19.tar.gz',
-                   'https://www.dropbox.com/s/qih3tjsir3xulrn/lucene-index-cord19-full-text-2020-05-19.tar.gz',
-                   'https://www.dropbox.com/s/7z8szogu5neuhqe/lucene-index-cord19-paragraph-2020-05-19.tar.gz'],
-    '2020-06-19': ['https://www.dropbox.com/s/bj6lx80wwiy5hxf/lucene-index-cord19-abstract-2020-06-19.tar.gz',
-                   'https://www.dropbox.com/s/vkhhxj8u36rgdu9/lucene-index-cord19-full-text-2020-06-19.tar.gz',
-                   'https://www.dropbox.com/s/yk6egw6op4jccpi/lucene-index-cord19-paragraph-2020-06-19.tar.gz']
+    '2020-05-19': ['https://www.dropbox.com/s/3ld34ms35zfb4m9/lucene-index-cord19-abstract-2020-05-19.tar.gz?dl=1',
```
Comment (rodrigonogueira4) [inline src/main/python/trec-covid/download_indexes.py:28]: why do we need `?dl=1?`

File: docs/experiments-covid.md [hunk 3]
```diff
@@ -81,19 +101,45 @@ They are, instead:
...
+tools/eval/trec_eval.9.0.4/trec_eval -c -M1000 -m recall.1000 -m ndcg_cut.10 -m map src/main/resources/topics-and-qrels/qrels.covid-round3.txt runs/anserini.final-r3.rf.post-processed.txt
+
+python tools/eval/measure_judged.py --qrels src/main/resources/topics-and-qrels/qrels.covid-round3.txt --cutoffs 10 --run runs/anserini.final-r3.fusion1.txt
+python tools/eval/measure_judged.py --qrels src/main/resources/topics-and-qrels/qrels.covid-round3.txt --cutoffs 10 --run runs/anserini.final-r3.fusion2.txt
+python tools/eval/measure_judged.py --qrels src/main/resources/topics-and-qrels/qrels.covid-round3.txt --cutoffs 10 --run runs/anserini.final-r3.rf.txt
+
+python tools/eval/measure_judged.py --qrels src/main/resources/topics-and-qrels/qrels.covid-round3.txt --cutoffs 10 --run runs/anserini.final-r3.fusion1.post-processed.txt
+python tools/eval/measure_judged.py --qrels src/main/resources/topics-and-qrels/qrels.covid-round3.txt --cutoffs 10 --run runs/anserini.final-r3.fusion2.post-processed.txt
+python tools/eval/measure_judged.py --qrels src/main/resources/topics-and-qrels/qrels.covid-round3.txt --cutoffs 10 --run runs/anserini.final-r3.rf.post-processed.txt
```
Comment (lukuang) [inline docs/experiments-covid.md:132]: Do we want to make this part of  `src/main/python/trec-covid/generate_round3_baselines.py` ?

Comment (lukuang) [reply docs/experiments-covid.md:132]: Maybe it can be done when you refactor the code

Comment (lintool) [reply docs/experiments-covid.md:132]: Yes, I was planning on doing it in a subsequent round.

File: src/main/python/trec-covid/download_indexes.py (hunk 2 above)
Comment (lintool) [reply src/main/python/trec-covid/download_indexes.py:28]: https://help.dropbox.com/files-folders/share/force-download

This forces download the actual file, as opposed to an HTML preview of the file.

File: docs/experiments-covid.md (hunk 1 above)
Comment (lintool) [reply docs/experiments-covid.md:12]: We explain below... and I think it makes the table look prettier ;)

**Pull request merged**